### ISSUE TITLE
Selection floating toolbar on post/comment properties

### ISSUE THREAD

**Issue by cocobelgica** [QuantumBadger/RedReader#526]
When selecting text in the post/comment properties popup dialog, the floating selection toolbar does not show the 3-dot "more" button to open the expansion menu for additional actions such us looking up the selected word using a dictionary app(given the app supports this), or other similar available actions. Screenshot below for reference.

![screenshot](http://i.imgur.com/kHE0KTH.jpg)

**Comment by QuantumBadger** [comment]
Thanks for the report. That popup toolbar is provided by Android, and while it looks like `setCustomSelectionActionModeCallback` might allow for customization, I can't see any standard API for enabling the three dots as pictured.

Do you get the three dots in any apps other than Chrome? (or whichever browser you're using in the screenshot?)

**Comment by cocobelgica** [comment]
Yes, just tested on a few apps and the 3 dots were there. For reference, I tested on MiXplorer's text editor, NewPipe's search box, and Simplenote.

**Comment by QuantumBadger** [comment]
> Yes, just tested on a few apps and the 3 dots were there. For reference, I tested on MiXplorer's text editor, NewPipe's search box, and Simplenote.

Thanks. I get the same three dots if I select some editable text in RedReader, for example in the Custom Location dialog. But the only option under there is "Share".

Do you get e.g. the dictionary option in all those other apps?

**Comment by cocobelgica** [comment]
Yes, I believe the custom actions(such as Dictionary, etc.)
are only available if there are other apps set to handle/process [ACTION_PROCESS_TEXT](https://medium.com/google-developers/custom-text-selection-actions-with-action-process-text-191f792d2999#.1itpgf75r).

**Comment by cocobelgica** [comment]
Found this tidbit from the Medium [article](https://medium.com/google-developers/custom-text-selection-actions-with-action-process-text-191f792d2999#.1itpgf75r) I linked in my previous comment:

> You may have already seen [documentation](http://developer.android.com/about/versions/marshmallow/android-6.0-changes.html#behavior-text-selection) and a [blog](http://android-developers.blogspot.com/2015/10/in-app-translations-in-android.html) post about ensuring the text selection toolbar and options appear in your app (in short: using a standard TextView/EditText and they’ll work out of the box, but note your EditText would need to have an android:id set and you need to call getDelegate().setHandleNativeActionModesEnabled(false) if you are using AppCompatActivity and want to use the native floating text selection toolbar on API 23+ devices).
